src/mergecil.ml

```diff
@@ -1298,24 +1334,18 @@
      (* Process a varinfo. Reuse an old one, or rename it if necessary *)
      let processVarinfo (vi : varinfo) (vloc : location) : varinfo =
        if vi.vreferenced then vi (* Already done *)
-       else if
-         (* Maybe it is static. Rename it then *)
-         vi.vstorage = Static
-         || (not !merge_inlines)
-            && hasAttribute "gnu_inline" (typeAttrs vi.vtype)
```

Comment:
So the `gnu_inline` attribute doesn't have anything to do with it at all?
Also, wasn't it even supposed to be `__gnu_inline__` according to GNU documentation?

Reply:
CIL removes the surrounding underscores around attributes.

> So the `gnu_inline` attribute doesn't have anything to do with it at all?

It does. If one wants to exactly follow the semantics of GCC, one would have to reject programs where multiple definitions of non-static inlines without the `gnu_inline` attribute appear. But since we are also more lenient for non-inline functions, I thought it made sense to also be more lenient here.